### DISCUSSION TITLE
SPACY and language issue (using pre trained model ) word2vec to Spacy Format for RASA

### DISCUSSION THREAD

**Discussion by jishtesting** [explosion/spaCy#7509] [Help: Coding & Implementations] [answered]
I’m trying to use a pre-trained model for RASA

Using word2vec I have converted the data to ZIP format But using the latest Spacy 3.0 version I am not able to convert it into Spacy usable format

**I tried python -m spacy init-model en ./spacy.word2vec.model --vectors-loc word2vec_shopping.txt.gz**

**word2vec_shopping.txt.gz** is the file name

![rasa cmd](https://user-images.githubusercontent.com/63588486/111896704-75469780-8a41-11eb-9ad8-1c59e9a31854.JPG)

This is the error I’m getting

**Error: No such command ‘init-model’.**

Please help with the new command for this.

**Comment by polm** [comment, marked as answer]
In v3 the functionality of the `init-model` command from v2 is now covered by `init`. 

https://spacy.io/api/cli#init-model

**Reply by jishtesting** [reply]
Thank you for the Response @polm 
I tried Replacing init-model with init 
_**python -m spacy init en spacy.word2vec.model --vectors-loc word2vec_shopping.txt.gz**_


but then this error

**Error: No such command 'en'.**

Can you please reframe the command If possible 

Just for your knowledge, these are the commands I have used 

**pip install spacy
python -m spacy download en_core_web_md**

**Reply by polm** [reply]
Converting your command it would look something like this:

    python -m spacy init vectors en word2vec_shopping.txt.gz output/

That will save output in the `output` directory.

Can you link to the RASA pretrained model you're trying to use, or the guide you're following?

**Reply by jishtesting** [reply]
@polm 
thank you for sharing the update command

the issue was Rasa 2.0 is not completely compatible with spacy 3.0 
when I downgraded the version to 2.3
it worked fine for me 


[https://forum.rasa.com/t/spacy-and-language-issue-using-pre-trained-model-word2vec-to-spacy-format/41522?u=jish](url)

This is  the link to the actual issue I was facing
